Orange/data/variable.py

```diff
@@ -744,25 +766,25 @@ def repr_val(self, val):
         """
         if isnan(val):
             return "?"
-        return '{}'.format(self.values[int(val)])
+        return '{}'.format(self._values[int(val)])
 
     str_val = repr_val
 
     def __reduce__(self):
         if not self.name:
             raise PickleError("Variables without names cannot be pickled")
         __dict__ = dict(self.__dict__)
-        __dict__.pop("values")
+        __dict__.pop("_values")
         return make_variable, (self.__class__, self._compute_value, self.name,
-                               self.values, self.ordered), \
+                               self._values, self.ordered), \
             __dict__
 
     def copy(self, compute_value=None, *, name=None, values=None, **_):
-        if values is not None and len(values) != len(self.values):
+        if values is not None and len(values) != len(self._values):
```

Comment:
Why was this change necessary? `self.values` being a property should still work right?

There are similar cases in other places as well...

Reply:
I can't decide. I changed it back.